fineract-core/src/main/java/org/apache/fineract/infrastructure/core/serialization/JsonParserHelper.java

```diff
@@ -518,17 +519,19 @@ public static LocalDate convertFrom(final String dateAsString, final String para
         return convertDateTimeFrom(dateAsString, parameterName, dateFormat, clientApplicationLocale).toLocalDate();
     }
 
-    public static LocalDateTime convertDateTimeFrom(final String dateTimeAsString, final String parameterName, final String dateTimeFormat,
+    public static LocalDateTime convertDateTimeFrom(final String dateTimeAsString, final String parameterName, String dateTimeFormat,
```

Comment:
`extractLocalDateTimeNamed` was missed. I reckon that method should be also enhanced.

Reply:
fixed